bluesky/plans.py

```diff
@@ -620,6 +620,30 @@ def reset():
     return (yield from finalize(plan_mutator(plan, insert_set), reset()))
 
 
+@contextmanager
+def baseline_context(plan_stack, devices, name='baseline'):
+    """
+    Read every device once upon entering and exiting the context.
+
+    The readings are designated for a separate event stream named 'baseline'
+    by default.
+
+    Parameters
+    ----------
+    plan_stack : iterable
```

Comment:
Nitpicking, but `iterable` isn't really right here. You could say `'list-like'` or something along those lines.